### PR TITLE
Fix Nexmo Client SDK Documentation

### PR DESCRIPTION
**Pull request by vonageviollete**
## Description
Fix the client SDK documentation for Nexmo.

### CODE REVIEW

File: _tutorials_tabbed_content/client-sdk/get-started/in-app-voice/in-app-calling-ios/call-type/objective-c.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ language: objective_c
 menu_weight: 2
 ---
 
-Note the second parameter in the `client?.call` method above - while `NXMCallHandlerInApp` is useful for simple calls, you can also start a call with customized logic [using a NCCO](/client-sdk/in-app-voice/concepts/ncco-guide), by choosing `NXMCallHandlerServer` as the `callHandler`.
+Note the second parameter in the `[self.nexmoClient call]` method above - while `NXMCallHandlerInApp` is useful for simple calls, you can also start a call with customized logic [using a NCCO](/client-sdk/in-app-voice/concepts/ncco-guide), by choosing `NXMCallHandlerServer` as the `callHandler`.
```
Comment (pardel) [inline _tutorials_tabbed_content/client-sdk/get-started/in-app-voice/in-app-calling-ios/call-type/objective-c.md]: "Note the second parameter in the `[self.nexmoClient call]` method above" should read:

"Note the second parameter in the `[self.nexmoClient call:callHandler:completionHandler:`]` method above"

In Objective-C, the method signature includes all the parts before each argument.

File: _tutorials_tabbed_content/client-sdk/get-started/in-app-voice/in-app-calling-ios/started-project/swift.md [hunk 2]
```diff
@@ -32,6 +32,18 @@ Using the Github project you cloned, in the Start folder, open `GettingStarted.x
...
-* `import NexmoClient` - imports the sdk
-* `let client = NXMClient.shared` - the NexmoClient shrared instance
-* `var call: NXMCall?` - property for the call instance
+
+* imports the SDK
+    ```swift
+    import NexmoClient
+    ```
+
+* the NexmoClient shrared instance
```
Comment (pardel) [inline _tutorials_tabbed_content/client-sdk/get-started/in-app-voice/in-app-calling-ios/started-project/swift.md]: shared is misspelt here.

**Comment by tbedford** [comment]
When attempting to test locally I get the following error:

```
~/violetta/nexmo-developer [master] $ yarn install
yarn install v1.19.0
[1/5] 🔍  Validating package.json...
error @1.1.0: The engine "node" is incompatible with this module. Expected version "> 8.0.0 <= 10". Got "12.11.1"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

However, I do **not** get this error with current NDP master. My other updated clones are also all working fine. I can only assume there is an error with the fork.

@fabianrbz are you able to test this fork successfully locally? 

I have added the Do not merge label until we can successfully test locally. Other option is to create a new PR off current master with the changes applied.

**Comment by tbedford** [comment]
I'm closing this PR as it seems to have been made on a an ancient fork of master. There have been significant reorganizations of the docs (especially client SDK docs) since this fork, and I'm not even able to test this PR locally in its current state. Please use the latest fork of master. Alternatively, give the changes required to the latest master directly to me and I will raise the PR. Many thanks.

**Pull request closed without merging**